DEVELOPER.md

```diff
@@ -260,9 +260,110 @@ Here are the steps on how to override them:
 
 4. Setup the commmunal endpoint.
 
+      1. AWS S3 BUCKET
+
+         If you have an MicroFocus account for AWS, you can configure your environment so that it uses AWS instead of minio.  There are pre-existing buckets setup for developers to use.  Here are the steps to set that up:
```

Comment:
Rather than say MicroFocus account, lets generalize it.  This is how we connect to any AWS endpoint.